earth_enterprise/src/server/pywms/ogc/wms_111/wmsxml/capabilities.py

```diff
@@ -31,22 +31,22 @@
     ) = range(3)
 XMLParser_import_library = None
 try:
-  # lxml
-  from lxml import etree as etree_
-  XMLParser_import_library = XMLParser_import_lxml
+  import defusedxml.ElementTree as etree_
```

Comment:
This code seems to be trying to find an XML library that works to import. Since we're forcing it to use defusedxml, we can probably remove lines 27-68 and replace it with a single `import defusedxml.ElementTree as etree_`. There's a couple other places where the code handles different libraries that we can likely clean up as well - basically, anything that refers to `XMLParser_import_*` is no longer needed.

The same applies to several of the other files on this PR.